config/default/user.role.authenticated.yml

```diff
@@ -12,4 +12,6 @@ permissions:
   - 'access content'
   - 'access shortcuts'
   - 'access site-wide contact form'
+  - 'view field_agency'
   - 'view media'
+  - 'view own field_agency'
```

Comment:
Do you know what these are/why they were added here?

Reply:
Well, I guess we could get rid of that as long as they have permission to `View anyone's value for field field_agency`